src/app/presentation/introjs-directive/introjs.directive.ts

```diff
@@ -8,12 +8,16 @@ import {introJs} from 'intro.js/intro.js';
 export class IntrojsDirective implements OnInit {
 
   constructor(public slide: SlideComponent) {
+    if (!localStorage.numTours) {
+      localStorage.numTours = 0;
+    }
   }
 
   ngOnInit() {
     this.slide.onActive.subscribe((active) => {
-      if (active) {
+      if (active && localStorage.numTours <= 2) {
```

Comment:
I think I use setItem to set the prop but Aziz showed me yesterday that object notation works too; I need to keep count of the tours so that the second exercise's tour still runs but no other tours after that.